assets/src/domain/eventEditor/hooks/entityListCacheUpdates/useUpdateEntityList.ts

```diff
@@ -0,0 +1,10 @@
+import { WriteQueryOptions } from '@edtrServices/apollo/queries';
+import { CacheUpdaterFn, useUpdateCache } from '@sharedServices/apollo/queries';
+
+const useUpdateEntityList = <List = any>(
+	writeQueryOptions: WriteQueryOptions<List> = undefined
+): CacheUpdaterFn<List> => {
+	return useUpdateCache<List>(writeQueryOptions);
+};
```

Comment:
can you plz add a comment to briefly explain what this does?

Is this for changing what entities are in the list? ie, via filtering? 
Or is this only used if you need to add a new entity or delete an existing one?

Reply:
This was already there, I just moved it inside its own directory. Will add explanatory comments.